cloud/pkg/edgecontroller/controller/downstream.go

```diff
@@ -425,6 +438,118 @@ func (dc *DownstreamController) syncEndpoints() {
 	}
 }
 
+// get service backend by service and its pod list, send backend instead of pod list to edge
+func getBackendByServiceAndPods(svc *v1.Service, pods []v1.Pod) *api.Backend {
+	if len(pods) == 0 {
+		klog.Warningf("service name: %s, namespace: %s with no pods", svc.Name, svc.Namespace)
+		return nil
+	}
+	bkd := &api.Backend{
+		Name:            svc.Name,
+		Namespace:       svc.Namespace,
+		ResourceVersion: svc.ResourceVersion,
+		HostNetwork:     pods[podIndex].Spec.HostNetwork,
```

Comment:
`HostNetwork` is assigned the network type of the first pod here？

Reply:
Yes, normally pods of the same deployment share the same network type.